templates/vue/src/components/NodeModal.vue

```diff
@@ -886,4 +918,8 @@ table {
 button:disabled {
   cursor: not-allowed;
 }
+
+.buttons-container > * {
+  margin: 0.25rem !important;
+}
```

Comment:
I suggest you use [bootstrap margin classes](https://getbootstrap.com/docs/4.3/utilities/spacing/) instead of adding a custom style.

Reply:
OK this one, I don't really want to change, as I would need to put a margin class on every single button under that div. Thoughts?

Also I don't think 0.25rem is one of the bootstrap defaults